src/main/resources/templates/catalog.html

```diff
@@ -108,9 +108,9 @@
                         <p class="search-none__one">Now your want list is empty.</p>
                         <p class="search-none__two">Useful tips:</p>
                         <ul class="search-none__ul-search">
-                            <li>You can add your discogs username and we will show vinyls according to your discogs want list</li>
-                            <li>You can add the record on the website <a href="https://www.discogs.com/" target="_blank">www.discogs.com</a> to the want list</li>
-                            <li>Or on our website click the star on the record</li>
+                            <li>You can <a href="/editProfile" class="underline-link">add your discogs username</a> and we will show vinyls according to your discogs want list</li>
+                            <li>You can add the record on the website <a href="https://www.discogs.com/" class="underline-link" target="_blank">www.discogs.com</a> to the want list</li>
+                            <li>Or <a href="/" class="underline-link">on our website</a> click the star on the record</li>
```

Comment:
Here should be ref to /catalog with parameter(for example), and server should return simple catalog page with 50 random vinyl. During demo you said that you come up with something for this case.